code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1761,8 +1761,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 							H.emote("painscream")
 						else
 							H.emote("pain")
-				if(damage_amount > ((H.STACON*10) / 3) && !HAS_TRAIT(H, TRAIT_NOPAINSTUN))
-					H.Immobilize(8)
+				if(damage_amount > ((H.STACON*12.5) / 3) && !HAS_TRAIT(H, TRAIT_NOPAINSTUN)) //We want this effect only on heavy hits.
```

Comment:
`(H.STACON * 3)` instead of `((H.STACON*12.5) / 3)` would make the numbers stay within an expected hit range, that number escalates to 60+ with 15 CON which is AFAIK literally infeasible to reach with any weapon atm